post.hbs

```diff
@@ -31,8 +21,28 @@
                             </span>
                         {{/foreach}}
                     </span>
+                    <span>
+                        Reading Time:&nbsp;
+                        {{reading_time}}
+                    </span>
                 </span>
-
+                
+                <span class="flex flex-auto content-center gap-2 justify-center"> 
+                    <div class="text-base">Tags:</div>
```

Comment:
Also is it possible to hide this part if there are no tags?

Reply:
Done